internal/anchor/reconciler.go

```diff
@@ -341,15 +342,20 @@ func (r *Reconciler) getInstance(ctx context.Context, pnm, nm string) (*api.Subn
 	return inst, nil
 }
 
-func (r *Reconciler) writeInstance(ctx context.Context, log logr.Logger, inst *api.SubnamespaceAnchor) error {
+func (r *Reconciler) writeInstance(ctx context.Context, log logr.Logger, inst, baseInst *api.SubnamespaceAnchor) error {
 	if inst.CreationTimestamp.IsZero() {
 		if err := r.Create(ctx, inst); err != nil {
 			log.Error(err, "while creating on apiserver")
 			return err
 		}
 	} else {
-		if err := r.Update(ctx, inst); err != nil {
-			log.Error(err, "while updating on apiserver")
+		err := r.Patch(ctx, inst, client.MergeFrom(baseInst))
+		if err != nil {
+			if apierrors.IsNotFound(err) {
```

Comment:
Was this needed for functionality or just for those (very rare) cases where it makes the logs messy?